src/Components/Onboarding/Wizard.tsx

```diff
@@ -28,7 +29,12 @@ class Wizard extends React.Component<Props, State> {
     }
 
     const CurrentStep = this.props.stepComponents[currentStep]
-    return <CurrentStep onNextButtonPressed={this.onNextButtonPressed.bind(this)} />
+    return (
+      <CurrentStep
+        onNextButtonPressed={this.onNextButtonPressed.bind(this)}
+        redirectTo={this.props.redirectTo}
+      />
```

Comment:
Cool -- not sure if this will work, but is it possible to avoid passing in `redirectTo` to each step and instead redirect from the Wizard [here](https://github.com/artsy/reaction/pull/495/files#diff-f494b90c7872d093afb16d77f131ae8fR28)? It seems like `getCurrentStep` can tell if we're at the last step. I'm sure his works too but I'm curious and it might allow us to not pass down `redirectTo` to every step.

Reply:
Oh yeah I think we talked about that implementation when we were first looking at this but I forgot about it. I'll give it a try.